feature/_fish/install.sh

```diff
@@ -1,23 +1,27 @@
 . /feature-installer-utils.sh
 
-FLAVOUR="X$(cat /home/cloudcontrol/flavour)X"
-if [[ "X${FLAVOUR}X" =~ X(azure|simple|tanzu|gcloud)X ]]
+if [[ "${FLAVOUR}" =~ (simple|tanzu|gcloud) ]]
 then
-  echo "Installing packages"
   execHandle 'Installing fish' sudo apk add fish perl fzf git
-elif [ "${FLAVOUR}" == "XawsX" ]
+elif [[ "${FLAVOUR}" == "azure" ]]
```

Comment:
did you omit the enclosing 'X...X' on purpose?
Same for the aws line below.

Reply:
Yes, I remembered that we had a talk about this at some point and decided that it might no be necessary anymore and as I was changing the scripts anyway I thought I'd directly optimize it as well.